js/src/tooltip.js

```diff
@@ -288,19 +281,24 @@ const Tooltip = (($) => {
 
         $(this.element).trigger(this.constructor.Event.INSERTED)
 
-        this._tether = new Tether({
-          attachment,
-          element         : tip,
-          target          : this.element,
-          classes         : TetherClass,
-          classPrefix     : CLASS_PREFIX,
-          offset          : this.config.offset,
-          constraints     : this.config.constraints,
-          addTargetClasses: false
+        this._popper = new Popper(this.element, tip, {
+          placement : attachment,
+          modifiers : {
+            offset : {
+              offset : this.config.offset
+            }
+          },
+          onCreate : (data) => {
+            if (data.originalPlacement !== data.placement) {
```

Comment:
It cause some issues because the Tooltip or Popover is already shown and our CSS apply some styles because of that Popper do not calculate the good position for our Tooltip/Popover